### ISSUE TITLE
[docs] Matrix transpose missing in formula in nn.functional.bilinear docs

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#36307]
It should say `x_2^T` if I'm not mistaken: https://github.com/pytorch/pytorch/commit/883628cb5c31c89323cef52e96310c92abecfcbe#r38390009

**Comment by vadimkantorov** [comment]
Same in torch.nn.Bilinear. Some transpose is missing somewhere https://pytorch.org/docs/stable/nn.html?highlight=bilinear#torch.nn.Bilinear

**Comment by choidongyeon** [comment]
I can fix this, but I have a clarification - shouldn't it be x_1^T A x_2? @vadimkantorov

**Comment by vadimkantorov** [comment]
@albanD For Bilinear, it'd be better to specify what exactly happens in batched case (will are dimensions be marginalized against, or just 1). Also, what does a product of `x^T A y` do when x is a matrix `(N1, H1)`, y is a matrix `(N2, H2)` and A is `(H, H1, H2)`. What does matrix product of x^T A mean? i.e. `(H1, N1)` product with `(H, H1, H2)`. Must N1 == N2? 

There is one formula in eq(1) in 
https://arxiv.org/pdf/1610.04325.pdf

**Comment by albanD** [comment]
Yes that documentation is not super clear. More work should be done there especially for higher dims.
